### ISSUE TITLE
Version 0.7 no config

### ISSUE THREAD

**Issue by terriblefox** [nicoh88/homebridge-xiaomi-roborock-vacuum#115]
I removed the old plugin and installed the new one with and without deleting the config. But the plugin settings are empty now.
<img width="847" alt="Bildschirmfoto 2020-02-26 um 16 38 56" src="https://user-images.githubusercontent.com/43913179/75361090-0d159880-58b7-11ea-9c67-0e7eeedeb5ac.png">

**Comment by c-o-m-m-a-n-d-e-r** [comment]
@afharo : i think there wasn't settings ... what do you say?

**Comment by afharo** [comment]
Sorry, I can't understand German. What is error message about? 
Also, what config are you talking about? Is the entire homebridge's config.json file empty?
Or do you mean the config-schema UI?
If so, can you right click, select "Inspect Element", then Console, and share any errors you may find in there?

**Comment by c-o-m-m-a-n-d-e-r** [comment]
oh i see, there comes a Internal Server Error (code 500)

<img width="984" alt="Bildschirmfoto 2020-02-26 um 21 22 30" src="https://user-images.githubusercontent.com/815885/75384294-25e57480-58de-11ea-8ec4-4b9e944f6235.png">

**Comment by afharo** [comment]
maybe because of this?
https://github.com/nicoh88/homebridge-xiaomi-roborock-vacuum/blob/master/config.schema.json#L51

The default value does not match the expected schema.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
i will test it right now ...

**Comment by terriblefox** [comment]
Just the UI scheme, thought there were some checkboxes for enable disable water tank etc... the robot is running normal... didn't checkout rooms so far

**Comment by afharo** [comment]
oh! I saw it! missing `:` after `"room"` 😅 @c-o-m-m-a-n-d-e-r

**Comment by c-o-m-m-a-n-d-e-r** [comment]
wow you are right, and i was wondering why my IDE was marking this section hehe.
Just a minute, will fix this and create a PR

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yes its the complete rooms block.
will take a deeper look whats exactly wrong there

**Comment by c-o-m-m-a-n-d-e-r** [comment]
@terriblefox : can you test it

npm install --save c-o-m-m-a-n-d-e-r/homebridge-xiaomi-roborock-vacuum#395a6f25a8f2833cf4691231543d233
5e9aecad9

its from my dev branche at the moment.

**Comment by terriblefox** [comment]
ill test

**Comment by nicoh88** [comment]
Successfully published homebridge-xiaomi-roborock-vacuum@0.7.1

```
sudo su -
npm -g install homebridge-xiaomi-roborock-vacuum@latest
```
